### DISCUSSION TITLE
cmp add a parenthesis when we call a jsx component

### DISCUSSION THREAD

**Discussion by guivictorr** [LazyVim/LazyVim#844] [Q&A] [answered]
hi, how can I disable this type of completition from cmp

![Screenshot 2023-05-28 at 02 53 45](https://github.com/LazyVim/LazyVim/assets/55333929/08a2b953-7c9c-4e76-a77b-a6ac0a9a6bd7)

**Comment by folke** [comment]
How did you resolve this? Not a LazyVim issue obviously, but I'm curious if there's a solution to this.

**Reply by guivictorr** [reply, marked as answer]
commenting this line this dont happen anymore 

![image](https://github.com/LazyVim/LazyVim/assets/55333929/6d4ef4f4-c56f-4792-9851-6e8cf111fcea)
